app/assets/javascripts/discourse/app/components/sidebar/user/section-link.js

```diff
@@ -25,7 +25,10 @@ export default class SectionLink {
   @bind
   didStartDrag(event) {
     // 0 represents left button of the mouse
-    if (event.button === 0) {
+    if (event.button === 0 || event.targetTouches) {
+      this.startMouseY = Math.round(
```

Comment:
You could put this `Math.round` logic in a private function like:

```js
#calcMouseY(event) {
  return Math.round(event.targetTouches ? event.targetTouches[0].clientY : event.y);
}
```

Reply:
It's repeated a few times in this component

Reply:
Nice one, thank you :)